### ISSUE TITLE
Access to the path C:\...\busybox is denied

### ISSUE THREAD

**Issue by gregamariani** [ClusterM/hakchi2#1146]
I am getting the access denied error using v2.21f and have attached the error log from the debug version.

I have already tried:
- disabling anti-virus (Windows Defender and VIPRE)
- deleting the hakchi-temp folder in AppData/Local/Temp
- unchecking the Read Only box under Properties in the main Hakchi folder. Although, when I uncheck>Apple>OK and then open Properties again, a black square remains in the box.

When I go into the hakchi-temp folder after it fails, and go into Properties > Security > Advanced, I am seeing several DENY permission records. I have tried deleting those which didn't help.

Any tips would be much appreciated.

[debuglog.txt](https://github.com/ClusterM/hakchi2/files/1596624/debuglog.txt)

**Comment by KMFDManic** [comment]
Did you delete the temp folder from hakchi directory?  

You can typically use the alternate version of hakchi2.  IE: If you have issues with Debug, use Standard.  If you use Standard, use Debug.  Disabling Virus protection doesn't always help.  It still tends to false positive due to clovershell.  You should be logged in as an Administrative Account.  Depending on which Operating System, this ability is sometimes inactive and disabled.

**Comment by gregamariani** [comment]
Thank you for the comment.

I just tried deleting the temp folder from the hakchi directory, on the debug version, tried to Flash the Custom Kernel and got the same issue.

There is no temp folder in the hakchi directory on the normal (non-debug) version. I tried it again on this version and got the same error.

When you say 'logged in as an Administrative Account', are you referring to the Administrator Account referenced [here](https://www.ghacks.net/2014/11/12/how-to-enable-the-hidden-windows-10-administrator-account/)?

I will try that.
